app/styles/vendor/material-theme.scss

```diff
@@ -117,6 +117,9 @@ mat-dialog-container.mat-dialog-container {
     width: auto !important;
     height: auto !important;
     max-height: 80vh;
+    // have the children control the overflow to get DDL to show in dialog footer.
+    // there might be another way but havent found it yet.
+    overflow: visible !important;
```

Comment:
hhm, yeah that one you should wait for select v2. That's one of the problem I was talking about

Reply:
This doesn't appear to affect anything else at this stage. As soon as select V2 goes in i am happy to remove this as its a bit of a dirty hack.

Reply:
removed this hack now as select changes have been merged in.